PlayerScripts/PlayerScriptsPlace/StarterPlayer/StarterPlayerScripts/ControlScript/MasterControl/TouchJump.rbxmx

```diff
@@ -97,6 +100,18 @@ function TouchJump:Create(parentFrame)
 		end
 	end)
 	
+	if humanoid then
+		
+		humanoid.Changed:connect(function(prop)
```

Comment:
My reasoning for using MasterControl was to avoid writing a similar function to get the humanoid in TouchJump, which I feel would have been redundant.